source/vscode/src/projectSystem.ts

```diff
@@ -225,6 +225,11 @@ async function singleFileProject(
   documentUri: vscode.Uri,
 ): Promise<IProjectConfig> {
   const file = await vscode.workspace.openTextDocument(documentUri);
+  // ToDo: the `getEntryProfile` doesn't acutally use the project loader object. Can I call it without it?
```

Comment:
It would be great if we didn't have to have the projectLoader loaded in order to use this function, as it doesn't really make use of it anyways. But I'm not sure right now how to get it to be exposed from the npm to the vscode without attaching it to an object like this.